docs/plan/model-prompt-caching-visibility.md

```
@@ -0,0 +1,75 @@
+2. 消息持久化 `token_usage` 扩展可选缓存字段（向后兼容）。
+3. 前端 token 统计 popover 展示缓存命中 token 数（i18n 双语）。
+
+### 非目标
+- xAI `x-grok-conv-id` 会话路由 header（需要调用链传会话 id，改动面大，收益低；单独评估）。
+- Gemini 1.5 显式 `cachedContent` 资源管理（需额外 API 生命周期管理，32k+ tokens 门槛；2.5+ 隐式缓存已覆盖）。
+- OpenAI 缓存写入计费（`cache_write_tokens`，仅 GPT-5.6+，当前无模型接入）。
+
+## High-Level Design
+
+### 数据流
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)